### PR TITLE
Updated cost docs

### PR DESCRIPTION
**Pull request by olonho**

### CODE REVIEW

File: runtime/runtime-params-estimator/src/wasmer_estimator.rs [hunk 1]
```diff
@@ -61,14 +62,32 @@ fn call() -> (Option<VMOutcome>, Option<VMError>) {
...
     let start = start_count(gas_metric);
     for _ in 0..NUM_ITERATIONS {
         call();
     }
-    let cost = end_count(gas_metric, &start);
-    cost / (outcome.burnt_gas * NUM_ITERATIONS)
+    let measured = end_count(gas_metric, &start);
+    // Those proportions led to the used formula:
+    //   outcome.burnt_gas = gas_burned(call)
+    //   measured = NUM_ITERATIONS * measured(call)
```
Comment (nearmax) [inline runtime/runtime-params-estimator/src/wasmer_estimator.rs:76]: We are conflating names here for `measured`, but be more readable to have two different names.

File: runtime/runtime-params-estimator/src/wasmer_estimator.rs [hunk 2]
```diff
@@ -61,14 +62,32 @@ fn call() -> (Option<VMOutcome>, Option<VMError>) {
...
+    //   measured(call) = num_x86_insns(call)
+    //   gas_burned(call) = gas_cost_per_wasm_op * num_wasm_ops(call)
+    //   gas_burned(call) = measured * GAS_IN_MEASURE_UNIT / DIVISOR
+    //   outcome.burnt_gas = num_wasm_ops(call) *
+    //       VMConfig::default().regular_op_cost
+    //   gas_cost_per_wasm_op = (measured * GAS_IN_MEASURE_UNIT *
+    //       VMConfig::default().regular_op_cost) /
+    //       (DIVISOR * NUM_ITERATIONS * outcome.burnt_gas)
```
Comment (nearmax) [inline runtime/runtime-params-estimator/src/wasmer_estimator.rs:84]: Not sure what `DIVISOR` is.

File: runtime/runtime-params-estimator/src/wasmer_estimator.rs [hunk 3]
```diff
@@ -61,14 +62,32 @@ fn call() -> (Option<VMOutcome>, Option<VMError>) {
...
     let outcome = outcome.unwrap();
     let start = start_count(gas_metric);
     for _ in 0..NUM_ITERATIONS {
         call();
     }
-    let cost = end_count(gas_metric, &start);
-    cost / (outcome.burnt_gas * NUM_ITERATIONS)
+    let measured = end_count(gas_metric, &start);
+    // Those proportions led to the used formula:
+    //   outcome.burnt_gas = gas_burned(call)
```
Comment (nearmax) [inline runtime/runtime-params-estimator/src/wasmer_estimator.rs:75]: Not sure how to read this. I suggest to clarify what is given and what is derived. To my understanding `outcome.burnt_gas` is a given, so they way to interpret `outcome.burnt_gas = gas_burned(call)` would be -- "this is how it is computed".

File: runtime/runtime-params-estimator/src/wasmer_estimator.rs [hunk 4]
```diff
@@ -61,14 +62,32 @@ fn call() -> (Option<VMOutcome>, Option<VMError>) {
...
+    //   measured = NUM_ITERATIONS * measured(call)
+    //   measured(call) = num_x86_insns(call)
```
Comment (nearmax) [inline runtime/runtime-params-estimator/src/wasmer_estimator.rs:77]: Not sure what this line means, is it just an FYI on how it is computed?

File: runtime/runtime-params-estimator/src/wasmer_estimator.rs (hunk 3 above)
Comment (olonho) [reply runtime/runtime-params-estimator/src/wasmer_estimator.rs:75]: Rewrote.

File: runtime/runtime-params-estimator/src/wasmer_estimator.rs (hunk 4 above)
Comment (olonho) [reply runtime/runtime-params-estimator/src/wasmer_estimator.rs:77]: Yes, exactly, so that it doesn't look too magical.

**Pull request merged**